database-commons/src/main/java/co/cask/DBManager.java

```diff
@@ -47,14 +47,15 @@ public void validateJDBCPluginPipeline(PipelineConfigurer pipelineConfigurer, St
     Preconditions.checkArgument(!(config.user == null && config.password != null),
```

Comment:
this class looks like utility class. jdbc driver lifecycle should be managed explicitly in plugins